webui/react/src/App.tsx

```diff
@@ -54,6 +55,7 @@ const AppView: React.FC = () => {
   return (
     <div className={css.base}>
       {isAuthenticated && <NavBar username={username} />}
+      <AppContexts />
```

Comment:
issue: will this run the `AppContexts` polling regardless of authentication state? Maybe something like this to ensure it's only executed during authenticated state:
```
{isAuthenticated && <AppContextx />}
```